Sources/Automerge/error.swift

```diff
@@ -13,6 +13,9 @@ typealias FfiReceiveSyncError = AutomergeUniffi.ReceiveSyncError
 // binding generator and we would rather not couple our public API to the
 // specific binding generator tech we're using
 
+/// An general document error.
+///
+/// The error is specific to the Rust language binding infrastructure.
```

Comment:
Is the purpose of the second paragraph here to explain why this isn't more descriptive (i.e. that it's not an enum with descriptive case names)?

Reply:
The way this'll show in DocC, the first line is the abstract that shows high level, and the second line will show in the reference page for this specific element as a second paragraph after the abstract. I've generally leaned on this location to provide additional detail for what the type is, does, or is used for beyond what's in the abstract - and repeating that these are all due to UniFFI/language bindings didn't seem relevant to the abstract itself.

I actually want to go back and re-read the uniffi generated bits to see how they're used to see if I can come up with better explanations of what they mean if they should appear, but thought this might be a reasonable placeholder to start with.